fusor-ember-cli/app/mixins/disable-tab-mixin.js

```diff
@@ -20,18 +20,15 @@ export default Ember.Mixin.create(ValidatesDeploymentNameMixin, {
   isValidDeploymentName: Ember.computed('model.name', 'deploymentNameValidator', function() {
     return this.get('deploymentNameValidator').isValid(this.get('model.name'));
   }),
-
-  // disable Next on Deployment Name if there is no deployment name
-  disableNextOnDeploymentName: Ember.computed.not('isValidDeploymentName'),
+  hasInvalidDeploymentName: Ember.computed.not('isValidDeploymentName'),
```

Comment:
Can we avoid the negative booleans? I know it's too late I just noticed it during a rebase and had a conflict in this file. The negative booleans make it more difficult to read in my opinion. I would rather have it be isValidDeploymentName then not it on the front end to get the negative.